Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -79,10 +79,14 @@ DECLARE_SOA_COLUMN(C1PtSnp, c1PtSnp, float, "fC1PtSnp");
 DECLARE_SOA_COLUMN(C1PtTgl, c1PtTgl, float, "fC1PtTgl");
 DECLARE_SOA_COLUMN(C1Pt21Pt2, c1Pt21Pt2, float, "fC1Pt21Pt2");
 
+// TRACKEXTRA TABLE definition
 DECLARE_SOA_COLUMN(TPCInnerParam, tpcInnerParam, float, "fTPCinnerP");
 DECLARE_SOA_COLUMN(Flags, flags, uint64_t, "fFlags");
 DECLARE_SOA_COLUMN(ITSClusterMap, itsClusterMap, uint8_t, "fITSClusterMap");
-DECLARE_SOA_COLUMN(TPCNCls, tpcNCls, uint16_t, "fTPCncls");
+DECLARE_SOA_COLUMN(TPCNClsFindable, tpcNClsFindable, uint8_t, "fTPCnclsFindable");
+DECLARE_SOA_COLUMN(TPCNClsFindableMinusFound, tpcNClsFindableMinusFound, uint8_t, "fTPCnclsFindableMinusFound");
```

Comment:
Are we sure Findable minus Found is always positive? If we have fake clusters, this might give negative values...

Reply:
I thought that findable is what is geometrically possible, and thus it does not matter if a cluster is real or fake, in this sense it should always be positive. @shahor02 ?

Reply:
@pzhristov Do you have negative cases in the Run 2 data which you are converting? We could add an assert in the conversion code in any case...

Reply:
Indeed, I would not rely on the assumption that it is always positive. I think the differences can be made signed int8_t, with condition that if its abs exceeds 127, it is set to 127.

Reply:
@shahor02 We could cover the full range by just shifting by (256-159) but this may be too messy to debug. I will implement it with int8_t then.

Reply:
ok, I would say such fine details should be tuned later.

Reply:
@pzhristov OK, it is committed. Can you adjust the converter?

Reply:
@jgrosseo Yes, I will modify the converter today.

Reply:
@pzhristov Let me know when you have adjusted the converter, then I'll merge this PR (or @ktf go ahead because I am offline this afternoon)